### PR TITLE
Update setup instructions for wizard

### PR DESCRIPTION
**Pull request by carise**

### CODE REVIEW

File: _docs/quickstarts/forseti-security/index.md [hunk 1]
```diff
@@ -4,30 +4,58 @@ order: 002
...
 ## Setting up Forseti Security
 
-The setup wizard presents a series of prompts to guide you through setting
-up a Forseti deployment, then generates a deployment template and configuration
-file based on your input.
+The setup wizard automatically determines setup information, generates a 
+deployment template, and creates a Forseti deployment.
+
+We recommend using Cloud Shell to run Forseti setup wizard to ensure 
+that you have the latest version of gcloud and the necessary gcloud components.
```
Comment (Tabrarian) [inline _docs/quickstarts/forseti-security/index.md]: I assume that "gcloud" refers to the gcloud command-line tool? All instances of "gcloud" should be updated for branding: please use either "Cloud SDK" or "gcloud command-line tool".

Also, this paragraph adds info that I don't really "need" to complete the quickstart. It's probably best to remove it in favour of a bullet in "Before you begin".

File: _docs/quickstarts/forseti-security/index.md [hunk 2]
```diff
@@ -4,30 +4,58 @@ order: 002
...
+Also, using Cloud Shell pre-authenticates you in your gcloud environment, 
+which is a necessary step for using the setup wizard.
+
+### Preliminary steps for setup without Cloud Shell
```
Comment (Tabrarian) [inline _docs/quickstarts/forseti-security/index.md]: Ideally, a quickstart is a quick way to get up and running without too many side paths. I think it would be best to skip this section and just focus on the Cloud Shell process.

File: _docs/quickstarts/forseti-security/index.md [hunk 3]
```diff
@@ -4,30 +4,58 @@ order: 002
...
 
-To complete this quickstart, you will need:
+Prior to running the setup wizard, you will need:
 
   - A GCP organization for which you want to deploy Forseti.
   - Org Admin IAM role in order for the script to assign the Forseti 
   service account roles on the organization IAM policy.
+  - A GCP project dedicated to Forseti.
+  - Enable billing on the project.
+
```
Comment (Tabrarian) [inline _docs/quickstarts/forseti-security/index.md]: Add a bullet:
`- The latest version of [Cloud SDK](https://cloud.google.com/sdk/downloads)`

File: _docs/quickstarts/forseti-security/index.md [hunk 4]
```diff
@@ -4,30 +4,58 @@ order: 002
...
+  1. Install [gcloud](https://cloud.google.com/sdk/downloads). If you already have 
+     gcloud, then update it.
+     
+  1. Ensure you have gcloud alpha components installed.
+  
+  1. Authenticate your user.
+     
+  1. Set your project.
+
+### Using Cloud Shell
```
Comment (Tabrarian) [inline _docs/quickstarts/forseti-security/index.md]:
```
### Activate Google Cloud Shell

1. Go to your [Cloud Platform Console](https://console.cloud.google.com/home/dashboard).
1. On the top bar, click **Activate Google Cloud Shell**.
```

File: _docs/quickstarts/forseti-security/index.md [hunk 5]
```diff
@@ -39,63 +67,28 @@ file based on your input.
...
-      
-      _e.g. Deploy 1.1.0:_
-      
-      ```bash
-      python setup_forseti.py --version 1.1.0
-      ```
-  
-     _Default: runs master branch_
      
       ```bash
```
Comment (Tabrarian) [inline _docs/quickstarts/forseti-security/index.md]: I agree with Michael, this section reads as if I should run all these commands together. Ideally they'd be separated into chunks with context for each command, something along these lines:

```
1. To start the setup process for the default release version, run `python setup_forseti.py`.
  2. To specify a Forseti release version, add the version flag `-- version #.#.#`.
To display help information, run `python setup_forseti.py -h`.
```

File: _docs/quickstarts/forseti-security/index.md [hunk 6]
```diff
@@ -39,63 +67,28 @@ file based on your input.
...
+      # Default: master branch
       python setup_forseti.py
-      ```
-      
-      _Run a certain branch:_
       
-      ```bash
+      # Specify a branch (e.g. develop)
```
Comment (Tabrarian) [inline _docs/quickstarts/forseti-security/index.md]: In what scenario would I need to specify a branch?

File: _docs/quickstarts/forseti-security/index.md [hunk 7]
```diff
@@ -39,63 +67,28 @@ file based on your input.
...
-  1. Setup wizard will create a Deployment Manager template based on your input
-  as well as ask whether you want to create the deployment.
-  1. Optionally create the bucket and a Cloud SQL instance. Only do this if you 
-  do not plan to create a deployment via Deployment Manager.
-  1. If you set up a service account to retrieve GSuite groups, follow the
-  post-setup instructions to enable your GSuite service account for domain-wide delegation.
-
-Forseti Security is now set up for your project and you can enable modules
-to use for your resources.
+  1. Setup will infer the necessary information to install Forseti. You will be
```
Comment (Tabrarian) [inline _docs/quickstarts/forseti-security/index.md]: Is this already supposed to work? I followed the steps here and I'm getting the old prompts to authenticate, choose organization, setup the project, etc.

Comment (carise) [inline _docs/quickstarts/forseti-security/index.md]: I just merged the changes to dev, so give it a try again.

```
cd forseti-security
git checkout dev
git pull
cd scripts/gcp_setup
python setup_forseti.py --branch dev
```

File: _docs/quickstarts/forseti-security/index.md (hunk 3 above)
Comment (carise) [inline _docs/quickstarts/forseti-security/index.md]: If the user is running setup in Cloud Shell, they will have Cloud SDK already.

Comment (Tabrarian) [inline _docs/quickstarts/forseti-security/index.md]: Gotcha, apologies; I'll close this one!

File: _docs/quickstarts/forseti-security/index.md [hunk 8]
```diff
@@ -39,63 +50,29 @@ file based on your input.
...
-  1. If you set up a service account to retrieve GSuite groups, follow the
-  post-setup instructions to enable your GSuite service account for domain-wide delegation.
-
-Forseti Security is now set up for your project and you can enable modules
-to use for your resources.
+  1. Setup will infer the necessary information to install Forseti. You will be 
+     prompted to enter a SendGrid API key, which is optional. (More information 
+     on setting up  [email notifications]({% link _docs/howto/configure/email-notification.md %}))
+     
+  1. You may be prompted to enter an ssh passphrase for Compute Engine. This will only
```
Comment (Tabrarian) [inline _docs/quickstarts/forseti-security/index.md]: If you previously used Cloud Shell to SSH to a Compute Engine instance and you set an SSH passphrase, setup prompts you to enter the passphrase. The Forseti setup uses secure copy (SCP) to copy the auto-generated G Suite service account key to the Forseti Compute Engine instance.

File: _docs/quickstarts/forseti-security/index.md [hunk 9]
```diff
@@ -39,63 +50,29 @@ file based on your input.
...
+  1. Setup will infer the necessary information to install Forseti. You will be 
+     prompted to enter a SendGrid API key, which is optional. (More information 
+     on setting up  [email notifications]({% link _docs/howto/configure/email-notification.md %}))
+     
+  1. You may be prompted to enter an ssh passphrase for Compute Engine. This will only 
+     happen if you've previously used Cloud Shell to ssh to a Compute Engine instance and
+     set a passphrase for ssh. The Forseti setup uses scp to copy the auto-generated G Suite 
+     service account key to the Forseti GCE instance.
+
+  1. After successfully running the setup and Deployment Manager, you should
```
Comment (Tabrarian) [inline _docs/quickstarts/forseti-security/index.md]: I'd rephrase slightly:

`After the setup wizard successfully completes Forseti setup and deployment, complete the steps to [enable G Suite Google Groups collection]({% link _docs/howto/configure/gsuite-group-collection.md %}).`

That said, is this a required step? In what scenario will I want to do this? Just noticed that's missing from the how-to as well. It would be great to provide some context for why a user would want to enable this.

Comment (carise) [inline _docs/quickstarts/forseti-security/index.md]: This really should be a required step, but most people who've run through setup (whether the wizard or manually) had trouble with the groups enablement due to 1) need to find the G Suite superadmin to handle the G Suite part for them, 2) they got stuck trying to figure out the G Suite/GCP UI (our screenshots should help now, but we haven't had any new deployments after updating the docs)

The setup wizard can't automate the groups enablement, so that's why we have the separate link.

**Comment by carise** [comment]
Thanks for the review!
